### PR TITLE
[#670] Render Rich Card Carousel (Google Style)…

### LINKED ISSUE #670: Render Rich Card Carousel (Google Style) on Live Chat Plugin
### Carousel cards 

- Carousel is limited to a minimum of 2 and a maximum of 10 cards
- The carousel controls ideally should be triggered by keyboard or click navigation
- Each card within the carousel should provide a button with a different option to be chosen

05 1 Rich cards – Carrousel@2x
05 2 Rich cards – Carrousel@2x
05 3 Rich cards – Carrousel@2x

### PR DESCRIPTION
**Pull request by bitboxer**
…on Live Chat Plugin

closes #670

### CODE REVIEW

File: frontend/chat-plugin/src/components/chat/index.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ let ws: WebSocket;
 
 const welcomeMessage: Message = {
   id: '19527d24-9b47-4e18-9f79-fd1998b95059',
-  content: JSON.stringify({text: 'Hello! How can we help you?'}),
+  content: {text: 'Hello! How can we help you?'},
```
Comment (bitboxer) [inline frontend/chat-plugin/src/components/chat/index.tsx:23]: This is a small fix to make sure the hello message can be rendered

**Comment by bitboxer** [comment]
I now combined the approach by @thorstenairy and mine and created a unified carousel scroll thing. It reacts totally dynamically and tries to fill the available width and only adds scroll buttons if needed. See here for an example:

https://user-images.githubusercontent.com/56195/108239333-1217cb80-714a-11eb-8938-e520f44094e8.mov

This also removes the downscaling of the templates for the chat plugin because we will increase the width in #1002 .

File: lib/typescript/render/components/Carousel/index.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
+import React, {useCallback, useEffect, useRef} from 'react';
+import styles from './index.module.scss';
+import {ReactComponent as LeftArrow} from 'assets/images/icons/leftArrow.svg';
+import {ReactComponent as RightArrow} from 'assets/images/icons/rightArrow.svg';
+
+export const Carousel = ({children}) => {
```
Comment (chrismatix) [inline lib/typescript/render/components/Carousel/index.tsx:6]: Lots of interesting learnings in this component. Thanks!

**Review by chrismatix — APPROVED** [review]
Amazing work!

**Pull request merged**